src/main/java/de/oliver/fancyholograms/commands/hologram/CenterCMD.java

```diff
@@ -0,0 +1,46 @@
+package de.oliver.fancyholograms.commands.hologram;
+
+import de.oliver.fancyholograms.FancyHolograms;
+import de.oliver.fancyholograms.api.hologram.Hologram;
+import de.oliver.fancyholograms.commands.Subcommand;
+import de.oliver.fancyholograms.util.Constants;
+import de.oliver.fancylib.MessageHelper;
+import org.bukkit.Location;
+import org.bukkit.command.CommandSender;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.List;
+
+public class CenterCMD implements Subcommand {
+    @Override
+    public List<String> tabcompletion(@NotNull CommandSender player, @Nullable Hologram hologram, @NotNull String[] args) {
+        return null;
+    }
+
+    @Override
+    public boolean run(@NotNull CommandSender player, @Nullable Hologram hologram, @NotNull String[] args) {
+        Location cloned = hologram.getData().getLocation().clone();
```

Comment:
`getLocation` returns a clone of the location so you don't need to run `clone` here

Reply:
Oh alright great